kalite/version.py

```diff
@@ -3,7 +3,7 @@
 # THIS IS USED BY settings.py.  NEVER import settings.py here; hard-codes only!
 MAJOR_VERSION = "0"
 MINOR_VERSION = "14"
-PATCH_VERSION = "dev2"
+PATCH_VERSION = "dev3"
```

Comment:
Because you can't overwrite existing uploads to PyPi, and I want to add more tests.